app/lib/features/authentication/signin/presentation/pages/sign_in_page.dart

```diff
@@ -21,45 +21,58 @@ class _LoginPageState extends State<LoginPage> {
   Widget build(BuildContext context) {
```

Comment:
could we use the previous design for now? since the design has been put in a lot of media 
- https://docs.google.com/presentation/d/1pDsYzExrFFQP_q2DauI_xN4U-P3DHXQGhKdMP3OfOVs/edit#slide=id.gc6fa3c898_0_10
- https://x.com/lwastuargo/status/1713385373441970555?t=xcmYW-UDLgyS_-jh3uttCg&s=08

let's redesign the entire app after the alpha release 🚀